x-pack/test/security_solution_api_integration/test_suites/detections_response/rules_management/prebuilt_rules/prebuilt_rule_customization/customization_enabled/import_rules.ts

```diff
@@ -159,27 +309,43 @@ export default ({ getService }: FtrProviderContext): void => {
               rule_source: { type: 'internal' },
               immutable: false,
             }),
-            expect.objectContaining({
-              rule_id: prebuiltRuleIds[0],
-              version: 1234,
-              rule_source: { type: 'external', is_customized: true },
-              immutable: true,
-            }),
             expect.objectContaining({
               rule_id: 'custom-rule-2',
               version: 1,
               rule_source: { type: 'internal' },
               immutable: false,
             }),
             expect.objectContaining({
-              rule_id: prebuiltRules[3].rule_id,
-              version: prebuiltRules[3].version,
+              rule_id: 'rule-1',
+              version: 2,
+              rule_source: { type: 'external', is_customized: true },
+              immutable: true,
+            }),
+            expect.objectContaining({
+              rule_id: 'rule-2',
+              version: 2,
               rule_source: { type: 'external', is_customized: false },
               immutable: true,
             }),
           ])
         );
       });
+
+      it('imports prebuilt rules when the rules package is not installed', async () => {
+        await deletePrebuiltRulesFleetPackage({ supertest, es, log, retryService }); // First we delete the rule package
+
+        const { body } = await importRules([prebuiltRules[0]]); // Then we import a rule which should cause the rule package to be redownloaded
+
+        expect(body).toMatchObject({
+          rules_count: 1,
+          success: true,
+          success_count: 1,
+          errors: [],
+        });
+
+        const status = await getPrebuiltRulesAndTimelinesStatus(es, supertest);
+        expect(status.rules_installed).toEqual(1); // The rule package is now redownloaded and recognizes the rule_id as an installed rule
```

Comment:
We should have a more elaborate set of assertions to make sure that the rule was imported correctly as a prebuilt rule.